src/pivotal-ui/javascripts/pivotal-ui.js

```diff
@@ -7,3 +7,4 @@ require('pui-prismjs');
 require('./scale')();
 require('@npmcorp/pui-react-back-to-top/jquery-plugin');
 require('@npmcorp/pui-css-drop-down-menu');
+require('../components/forms');
```

Comment:
Without specific `require` being called on any JS, it won't run into issues.

In this context it will be whatever the `main` attribute refers to in the `package.json` file in `src/pivotal-ui/components/forms`